lib/pengine/group.c

```diff
@@ -101,10 +101,19 @@ group_unpack(pe_resource_t * rsc, pe_working_set_t * data_set)
     xmlNode *xml_obj = rsc->xml;
     xmlNode *xml_native_rsc = NULL;
     group_variant_data_t *group_data = NULL;
+
+    // @COMPAT: group_ordered is deprecated since 2.1.5
```

Comment:
The task said that the `ordered` meta attribute is undocumented for groups and can be deprecated. I noticed it's also used for clones and bundles, and it seems to be undocumented for those cases too. Did we want to leave it there or remove it? That's why I didn't deprecate the constant yet.

Reply:
Those are still valid (and should be documented). It's essentially a kind=Serialize ordering for clone/bundle instances.

Reply:
`ordered` is documented for clones, at least. I just remembered we've used that for GFS2 for years. I was Ctrl+F-ing for `"ordered"` with quotes, looking for an XML snippet in Pacemaker Explained.

It looks like it might be undocumented for bundles though.

Incidentally, the `ordered` meta attribute actually **was** documented for groups: https://clusterlabs.org/pacemaker/doc/2.1/Pacemaker_Explained/singlehtml/#group-options

Reply:
With that in mind, it still looks like a good use case for sets. So I think the best approach is to remove it from the documentation. Let me know if you agree with that

Reply:
Yep